install.sh

```diff
@@ -385,6 +440,13 @@ install_apk() {
 }
 
 install_standalone() {
+	# Check if the unzip package is installed. If not error peacefully.
+	if !(command_exists unzip); then
+		echoh
+		echoerr "This script needs the unzip package to run."
+		echoerr "Please install unzip to use this function"
+		exit 1
+	fi
```

Comment:
I don't believe unzip is needed on all platforms, right?

Reply:
Only darwin needs it:
<img width="1485" alt="image" src="https://user-images.githubusercontent.com/7122116/210680663-6a079011-fd27-4f3e-b924-54e71c26b107.png">

Reply:
I can remove that part if not using darwin in the standalone function, but for the terraform install section it is needed. Some linux distros, somehow, don't have it preinstalled and it is needed as the terraform binaries come in zip files.

Reply:
Yup, makes sense.